src/content/docs/en/installation/rpi.mdx

```diff
@@ -13,24 +13,34 @@ Ultramarine Linux supports the Raspberry Pi 3 and 4. Please see the [Anywhere pa
 
 The drive you use deeply affects the experience you will have on Ultramarine.
 
-### Raspberry Pi 3
+### Raspberry Pi 3 and Zero 2
 
-Raspberry Pi 3 supports only microSD cards. Your storage device must be 16GB or larger.
+These models support only microSD cards. Your storage device must be 16GB or larger.
```

Comment:
"must"? "should"?

Reply:
we can bring it back now that i know the images are smaller